Zend/zend_language_parser.y

```diff
@@ -1114,6 +1116,8 @@ expr:
 			{ $$ = zend_ast_create_binary_op(ZEND_IS_EQUAL, $1, $3); }
 	|	expr T_IS_NOT_EQUAL expr
 			{ $$ = zend_ast_create_binary_op(ZEND_IS_NOT_EQUAL, $1, $3); }
+	|	expr T_PIPE expr
+			{ $$ = zend_ast_create(ZEND_AST_CALL, $3, zend_ast_create_list(1, ZEND_AST_ARG_LIST, $1) ); $$->attr = ZEND_CALL_SYNTAX_PIPE; }
```

Comment:
@nikic Is there something we could/should do with `CG(zend_lineno)` here? I don't know how the global is really used, so I don't know if this would have any positive impact, or if it should be elsewhere on the line (and I didn't try compiling this):

```
	|	expr T_PIPE { $<num>$ = CG(zend_lineno); } expr
			{ $$ = zend_ast_create(ZEND_AST_CALL, $4, zend_ast_create_list(1, ZEND_AST_ARG_LIST, $1) ); $$->attr = ZEND_CALL_SYNTAX_PIPE; }
```

Was trying to think of how we can make it as helpful as possible for debuggers, profilers, etc to have accurate line numbers.